package.json

```diff
@@ -32,12 +32,13 @@
 		"./dist/esm/lib/refs.node.js": "./dist/esm/lib/refs.browser.js"
 	},
 	"scripts": {
-		"build": "rimraf dist && npm run compile && npm run build:esm && npm run build:cjs && npm run build:declarations && npm run build:cp",
+		"build": "rimraf dist && npm run compile && npm run build:esm && npm run build:cjs && npm run build:declarations && npm run build:cp && npm run build:post",
 		"build:esm": "tsc --module es2015 --outDir ./dist/esm",
 		"build:cjs": "tsc --module commonjs --outDir ./dist/cjs",
 		"build:declarations": "tsc --module es2015 --outDir ./dist/esm --declaration true --allowJs false",
 		"build:watch": "npm run compile && npm run build:cp && tsc --watch --module es2015 --outDir ./dist/esm",
```

Comment:
I think you gotta do a `build:post` here too, otherwise `:watch` won't generate a `build.json`.

Reply:
ill handle that in another pr - there are also vpweb changes needed. using tsc-watch

Reply:
fair enough!